code/game/gamemodes/traitor/traitor.dm

```diff
@@ -8,7 +8,7 @@
 	name = "traitor"
 	config_tag = "traitor"
 	restricted_jobs = list("Cyborg","Mobile MMI")//They are part of the AI if he is traitor so are they, they use to get double chances
-	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain")//AI", Currently out of the list as malf does not work for shit
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "AI")//AI", Currently out of the list as malf does not work for shit
```

Comment:
Might want to remove that comment then.

Reply:
I wanted it so we know for posterity how lazy oldcoders were but I can do that